src/test/scala/fr/polytechnique/cmap/cnam/etl/extractors/hospitalstays/HospitalStayExtractorSuite.scala

```diff
@@ -4,7 +4,7 @@ package fr.polytechnique.cmap.cnam.etl.extractors.hospitalstays
 
 import org.apache.spark.sql.Dataset
```

Comment:
You will need to rename the whole class.

Reply:
Done